contrib/completion/bash/docker

```diff
@@ -3698,7 +3698,7 @@ _docker_swarm_init() {
 			COMPREPLY=( $( compgen -W "active drain pause" -- "$cur" ) )
 			return
 			;;
-		--cert-expiry|--default-addr-pool|--default-addr-pool-mask-length|--dispatcher-heartbeat|--external-ca|--max-snapshots|--snapshot-interval|--task-history-limit )
+		--cert-expiry|--data-path-port|--default-addr-pool|--default-addr-pool-mask-length|--dispatcher-heartbeat|--external-ca|--max-snapshots|--snapshot-interval|--task-history-limit )
```

Comment:
Are we missing `--data-path-addr` here? (if so, we should have a separate PR for that first)

@albers could you check if it has to be added here?

Reply:
This is the correct location for an option with an argument for which we cannot not provide a specific completion.

Reply:
Thanks! I can open a PR to add it 👍

Reply:
oh, actually, I see it's a few lines below